services/oc-pmrmrg/src/pmr_mrg/run_model.py

```diff
@@ -0,0 +1,96 @@
+import OpenCOR
```

Comment:
probably good to add a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top of file such as:
```
#!/usr/bin/env python3
```